package/CHANGELOG

```diff
@@ -13,28 +13,30 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
-??/??/16 kain88-de, jbarnoud
+??/??/16 tyler.je.reddy, kain88-de, jbarnoud, richardjgowers
 
-  * 0.13.1
+  * 0.14.0
```

Comment:
If I misunderstood where this change came from (eg from @richardjgowers 's merge) then apologies (I typically do rebases against develop on these branches).